src/main/scala/org/broadinstitute/dsde/snoop/ws/SnoopApiFormats.scala

```diff
@@ -5,6 +5,17 @@ import spray.json._
 import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
 import scala.annotation.meta.field
 
+/**
+ * Marker trait for normal messages
+ */
+trait RestMessage
+
+/**
+ * Message used to signify that a validation error happened
+ * @param message
+ */
+case class Validation(message: String)
```

Comment:
Why does "RestMessage" have Message but Validation not (or vice versa)? I think the two should be consistently named.

Also I like having some base trait for messages for a particular actor class, e.g. 'sealed trait FooActorMessage' and then extending that base on everything. Admittedly it doesn't have much value add but I like enjoy types